megamek/src/megamek/client/ui/swing/boardview/EntitySprite.java

```diff
@@ -860,6 +865,20 @@ else if (numAdv > 1)
                 addToTT("InfSpec", BR, Infantry.getSpecializationName(spec));
             }
         }
+        
+        //add portrait?
+        String category = entity.getCrew().getPortraitCategory(0);
```

Comment:
Is `getPortraitXXX(0)` valid when you hover over ejected units? And are all units crewed?

Reply:
Yes, I just checked and the portrait goes with the ejected pilot. I have added a check for null crews to avoid any NPEs.

Reply:
I don't see the check for null crews.